freestyle/shared/src/test/scala/freestyle/implicits.scala

```diff
@@ -53,7 +53,7 @@ class implicitsTests extends WordSpec with Matchers {
 
     "provide a custom implicit not found message for a missing Capture instance" in {
       shapeless.test.illTyped(
-        """SCtors1[SCtors1.Op].x(1).exec[scala.util.Try]""",
+        """SCtors1[SCtors1.Op].x(1).freeS.exec[scala.util.Try]""",
```

Comment:
The test messages (`"provide a custom ...`) of this test and the previous one also seem wrong, I think they need to be swapped to make sense.

Maybe we can fix this in this PR?.